docs/blog/articles/2025-05-27-KServe-0.15-release.md

```diff
@@ -1,4 +1,4 @@
-# Announcing KServe v0.15: Advancing Generative AI Model Serving
+# Announcing KServe v0.15:Advancing Generative AI Model Serving
```

Comment:
This is unintentional change?

Reply:
something is wrong with our spacing https://kserve.github.io/website/0.15/blog/articles/2025-05-27-KServe-0.15-release/

Reply:
That was fixed in https://github.com/kserve/website/pull/486

Reply:
I have merged to v0.15 branch

Reply:
ok will revert